.github/workflows/update_localization.yml

```
@@ -0,0 +1,48 @@
+
+jobs:
+  run:
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout repository
+        uses: actions/checkout@v4
+      - name: download previous translation template from successful CI run
+        shell: bash
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
```

actionlint

shellcheck reported issue in this script: SC2086:info:8:17: Double quote to prevent globbing and word splitting

(shellcheck)